openapi/src/parts/alpha1.yaml

```diff
@@ -37,6 +83,25 @@ paths:
 
 components:
   schemas:
+    LoadSignedUrlListRequestBody:
+      description: |
+        Request to load signed urls files to a gcs bucket.
+      type: object
+      properties:
+        jobControl:
+          $ref: '#/components/schemas/JobControl'
+        urlList:
+          description: url (http or https only) to the tsv file that contains the list of files.
```

Comment:
oh.. i didn't realize that's not what signed urls are :) how are we suppose to access non public urls?

Reply:
I am hoping that the documentation is wrong there and that with a signed URL, it will work.
The signed URL encodes some kind of token, valid for a limited time, that allows anyone holding the URL to access the file. I don't know the specifics on GCP.

Here's a link
https://cloud.google.com/storage/docs/access-control/signed-urls

Reply:
Well signed URLs are public URLs, so the manifest would need to contain a list of signed URLs (or otherwise public urls).

Reply:
but +1 to something like manifest url.